homeassistant/components/webostv/__init__.py

```diff
@@ -62,65 +49,27 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-CALL_SCHEMA = vol.Schema({vol.Required(ATTR_ENTITY_ID): cv.comp_entity_ids})
-
-BUTTON_SCHEMA = CALL_SCHEMA.extend({vol.Required(ATTR_BUTTON): cv.string})
-
-COMMAND_SCHEMA = CALL_SCHEMA.extend({vol.Required(ATTR_COMMAND): cv.string})
-
-SERVICE_TO_METHOD = {
-    SERVICE_BUTTON: {"method": "async_button", "schema": BUTTON_SCHEMA},
-    SERVICE_COMMAND: {"method": "async_commmand", "schema": COMMAND_SCHEMA},
-}
-
 _LOGGER = logging.getLogger(__name__)
 
 
 async def async_setup(hass, config):
     """Set up the LG WebOS TV platform."""
     hass.data[DOMAIN] = {}
 
-    tasks = [async_setup_tv(hass, config, conf) for conf in config.get(DOMAIN, [])]
+    tasks = [async_setup_tv(hass, config, conf) for conf in config[DOMAIN]]
     if tasks:
         await asyncio.gather(*tasks)
 
-    async def async_service_handler(service):
-        method = SERVICE_TO_METHOD.get(service.service)
-        params = {
-            key: value for key, value in service.data.items() if key != "entity_id"
-        }
-        entity_ids = service.data.get(ATTR_ENTITY_ID)
-        target_players = []
-        for entry in hass.data[DOMAIN].values():
-            player = entry["media_player"]
-            if entity_ids == ENTITY_MATCH_ALL or player.entity_id in entity_ids:
-                target_players.append(player)
-
-        calls = []
-        for player in target_players:
-            calls.append(getattr(player, method["method"])(**params))
-
-        if calls:
-            await asyncio.gather(*calls)
-
-    for service in SERVICE_TO_METHOD:
-        schema = SERVICE_TO_METHOD[service]["schema"]
-        hass.services.async_register(
-            DOMAIN, service, async_service_handler, schema=schema
-        )
-
     return True
 
 
 async def async_setup_tv(hass, config, conf):
     """Set up a LG WebOS TV based on host parameter."""
 
-    host = conf.get(CONF_HOST)
-    standby_connection = conf.get(CONF_STANDBY_CONNECTION)
-    config_file = hass.config.path(conf.get(CONF_FILENAME))
-
-    client = WebOsClient(host, config_file, standby_connection=standby_connection)
+    host = conf[CONF_HOST]
+    standby_connection = conf[CONF_STANDBY_CONNECTION]
 
+    client = WebOsClient(host, standby_connection=standby_connection)
```

Comment:
What is the default location of the config file?

Reply:
The client library uses a single config file for multiple clients (with clients identified by hostname/ip address).  The default location is $HOME/.aiopylgtv, but if $HOME is not set, it falls back to <os.getcwd()>/.aiopylgtv so I think it should be fine.

Reply:
Will it also work on docker?

Reply:
I restored the previous default behaviour just to be sure (config saved to webostv.conf in the home assistant configuration directory)